src/forms/CurrentIncome.js

```diff
@@ -66,54 +66,63 @@ const IncomeForm = function ({ current, time, setClientProperty }) {
       <IntervalColumnHeadings type={ type } />
 
       {/* All kinds of things need to be explained. */}
-      {/* @todo Change 'labelInfo' to visible blurb at top */}
+      
       <CashFlowRow
         { ...sharedProps }
-        generic='earned'
-        labelInfo='(Weekly income = hourly wage times average number of work hours per week)'>
-          Earned income
+        generic='earned'>
+        { snippets.earnedIncome.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='TAFDC'> TAFDC
+        generic='TAFDC'> 
+        { snippets.TAFDC.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='SSI'> SSI
+        generic='SSI'> 
+        { snippets.SSI.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='SSDI'> SSDI
+        generic='SSDI'>
+        { snippets.SSI.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='childSupportIn'> Child support received
+        generic='childSupportIn'>
+        { snippets.childSupport.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='unemployment'> Unemployment
+        generic='unemployment'> 
+        { snippets.unemployment.label }      
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='workersComp'> Worker’s comp
+        generic='workersComp'> 
+        { snippets.workersComp.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='pension'> Pension
+        generic='pension'>
+        { snippets.pension.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='socialSecurity'> Social security
+        generic='socialSecurity'>
+        { snippets.socialSecurity.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='alimony'> Alimony
+        generic='alimony'> 
+        { snippets.alimony.label }
       </CashFlowRow>
       <CashFlowRow
         { ...sharedProps }
-        generic='otherIncome'> Other income
+        generic='otherIncome'>
+        { snippets.otherIncome.label }
       </CashFlowRow>
-      <Form.Field>This prototype will attempt to make its own calculations for SNAP amount</Form.Field>
+      <Form.Field>{ snippets.bottomText }</Form.Field>
```

Comment:
Apparently I forgot to post a comment on this - we can possibly change this later, but `bottomText` isn't very descriptive. What if we add text under it? How about something talking about 'snap'? I figure there's some time to fiddle with it because I'd rather wait till the `ManagedNumberField` error we introduced earlier is all sorted.